airflow/providers/celery/executors/celery_executor_utils.py

```diff
@@ -60,16 +60,26 @@
 
     TaskInstanceInCelery = Tuple[TaskInstanceKey, CommandType, Optional[str], Task]
 
+
+def _get_celery_configuration():
```

Comment:
As @uranusjr proposed, can you extract `_get_celery_app` method instead. Also instead of running provider configuration manually, just decorate the method with `@provider_configuration_loaded` decorator similarly to `worker` celery command.